amivapi/settings.py

```diff
@@ -65,6 +65,7 @@
 
 # SMTP server defaults
 API_MAIL = 'api@amiv.ethz.ch'
```

Comment:
Do we want to change this default to `info@amiv.ch`? We had users actually emailing to the api email when something with their eventsignups went wrong, and the API admins are definitely not the right people to handle this. It is confusing to users that you can answer and then the email does not go to people responsible for the event, but the developers of the API.

Reply:
Good point. But this can be done in the settings as well, so I think changing the default is not a priority. For that I guess we first need a discussion which email is the appropriate one.

For that, we might include a 'reply-to' field.

Reply:
ok my comment jsut became the review summary, anyways: 

Yes, `reply-to` would actually be what we want. Please add it to the config here.